### DISCUSSION TITLE
可以做成插件吗？

### DISCUSSION THREAD

**Discussion by SK-Luffa** [he3-app/he3-feedback#8] [Ideas]
我有一个想法，如果这么强大的功能可以做成一个插件的话，（例如：vscode插件，chrome插件 甚至不需要全部完成的功能）那么感觉这将会是非常棒的😊

**Comment by zhenweiwang1990** [comment]
暂时还没有做插件的计划，但这确实是一个很不错的想法，我们内部评估一下。
